packages/link/src/__demo__/examples/states.js

```diff
@@ -13,27 +13,26 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import darken from 'polished/lib/color/darken';
```

Comment:
Should we be concerned that git apparently thinks this file is a renamed `packages/site/src/components/Link.js`?